### PR TITLE
Fix for mac mousewheel.

### PR DESCRIPTION
**Pull request by akallabeth**

### CODE REVIEW

**Comment by akallabeth** [comment]
@kjohnstoncbt could you please run `clang-format` on your changes and push that?

**Comment by kjohnstonadept** [comment]
> @kjohnstoncbt could you please run `clang-format` on your changes and push that?

Will do!

**Comment by akallabeth** [comment]
@freerdp-bot test

File: client/Mac/MRDPView.m [hunk 1]
```diff
@@ -548,7 +551,8 @@ - (void)keyDown:(NSEvent *)event
 	         "keyDown: keyCode: 0x%04X scancode: 0x%04X vkcode: 0x%04X keyFlags: %d name: %s",
 	         keyCode, scancode, vkcode, keyFlags, GetVirtualKeyName(vkcode));
 #endif
-	sync_keyboard_state(instance);
+    // See https://github.com/FreeRDP/FreeRDP/issues/4769
+    //sync_keyboard_state(instance);
```
Comment (akallabeth) [inline client/Mac/MRDPView.m:555]: So, missing formatting and what does this have to do with the mouse wheel?

**Comment by kjohnstonadept** [comment]
That was pushed in error - sorry I will undo that.

**Comment by akallabeth** [comment]
@freerdp-bot test
